data/strings/sound.txt

```diff
@@ -1567,7 +1604,7 @@
     fa = در پنجاه قدمی
     fi = Viidenkymmenen jalan päässä
     fr = Dans cinquante pieds
-    hi = पचास फीट में।
+    hi = पचास फीट में
```

Comment:
I'm not familiar with Hindi, but is it ok here and everywhere in PR for _hi_ ?

Reply:
Yes the `।` character is a period in Hindi, so that has been removed and is the only change.

`पचास फीट में। दायीं ओर मुड़ें। पर highway 123` is translated as `In fifty feet. Turn right. on highway 123`

`पचास फीट में दायीं ओर मुड़ें पर highway 123` is translated as `turn right in fifty feet onto highway 123`